skins/oasis/css/core/breakpoints-background.scss

```diff
@@ -178,3 +178,22 @@ body.skin-oasis {
 		@include background-max;
 	}
 }
+
+body.skin-oasis.uap-skin {
+	$uap-color: mix(white, $color-body, 15%);
+	@if (luminance($color-body) > 0.3) {
+		$uap-color: mix(black, $color-body, 10%);
```

Comment:
And we can try with [Mr. Black](https://github.com/Wikia/app/blob/dev/extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/_colors.scss#L33)